nucypher/blockchain/eth/clients.py

```diff
@@ -152,7 +152,7 @@ def peers(self):
     def chain_name(self) -> str:
         if not self.is_local:
             return PUBLIC_CHAINS[int(self.chain_id)]
-        name = LOCAL_CHAINS.get(self.chain_id, UNKNOWN_DEVELOPMENT_CHAIN_ID)
+        name = LOCAL_CHAINS.get(int(self.chain_id), UNKNOWN_DEVELOPMENT_CHAIN_ID)
```

Comment:
we can probably remove the casting to `int`. `chain_id` was updated to return an `int`, no? Similar for the line above (154).